systems/controllers/osc/operational_space_control.cc

```diff
@@ -561,10 +568,10 @@ VectorXd OperationalSpaceControl::SolveQp(
     if (tracking_data->IsActive() &&
         time_since_last_state_switch >= t_s_vec_.at(i) &&
         time_since_last_state_switch <= t_e_vec_.at(i)) {
-      VectorXd ddy_t = tracking_data->GetYddotCommand();
-      MatrixXd W = tracking_data->GetWeight();
-      MatrixXd J_t = tracking_data->GetJ();
-      VectorXd JdotV_t = tracking_data->GetJdotTimesV();
+      const VectorXd& ddy_t = tracking_data->GetYddotCommand();
```

Comment:
Do you know if this makes a shallow reference? I tried searching online on how to make a shallow copy, which is what I think we want here but wasn't able to find anything other than using the `auto` keyword which is not recommended.

Reply:
Do you mean we should use a copy instead of reference?

Reply:
I use reference here to avoid extra memory allocation for the temporary copy

Reply:
Okay so this doesn't make an extra copy - yeah that is what we want. I was just making sure that this avoid the extra copy.